graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -389,6 +389,23 @@ def test_roundtrip_with_arrow_22(gds: GraphDataScience) -> None:
         G_2.drop()
 
 
+@pytest.mark.encrypted
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
+def test_roundtrip_with_arrow_encrypted(gds_with_tls: GraphDataScience) -> None:
+    G, _ = gds_with_tls.graph.project(GRAPH_NAME, {"Node": {"properties": ["x", "y"]}}, {"REL": {"properties": "relX"}})
+
+    rel_df = gds_with_tls.graph.streamRelationshipProperty(G, "relX")
+    node_df = gds_with_tls.graph.streamNodeProperty(G, "x")
```

Comment:
Shouldn't we use the new, renamed endpoints here? Or maybe if we do then it won't work on 2.1 so probably not